src/payment/strategies/klarnav2/klarnav2-payment-strategy.ts

```diff
@@ -18,6 +18,7 @@ import KlarnaV2ScriptLoader from './klarnav2-script-loader';
 export default class KlarnaV2PaymentStrategy implements PaymentStrategy {
     private _klarnaPayments?: KlarnaPayments;
     private _supportedEUCountries = ['AT', 'DE', 'DK', 'FI', 'GB', 'NL', 'NO', 'SE', 'CH'];
+    private _supportedOCCountries = ['AU', 'NZ'];
```

Comment:
if this is always the same as klarna v1, could you extract these values in a different file so they are in sync?